packages/api-explorer/src/CodeSample.jsx

```diff
@@ -68,7 +68,7 @@ class CodeSample extends React.Component {
               <div key={key} style={{ display: selected ? 'block' : 'none' }}>
                 <CopyCode key={`copy-${key}`} code={example.code} />
                 <pre className="tomorrow-night tabber-body" style={{ display: selected ? 'block' : '' }}>
-                  {syntaxHighlighter(example.code, example.language, { dark: true })}
+                  {syntaxHighlighter(example.code, example.language, { customTheme: 'tomorrow-night' })}
```

Comment:
`@readme/syntax-highlighter` no longer natively supports the `tomorrow-night` theme that we use for code samples so we're passing this in. CSS for the theme is already being loaded within the explorer and our Hub2 bundle in prod so we don't need to worry about that.